vendor/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+  - childsb  # Storage
+  - knobunc  # Network, Multi-cluster, Storage
```

Comment:
Need to sync with architecture team on this, give me a bit.

Reply:
ping?